services/sentry/lib/index.js

```diff
@@ -0,0 +1,7 @@
+function channelExhausted(channel) {
+	return (
+		channel.exhausted && Object.values(channel.exhausted)[0] && Object.values(channel.exhausted)[1]
```

Comment:
isn't this an array? the default mongo behavior is to convert numeric indexes to arrays, so then you can check `return channel.exhausted.every(x => x)`

Reply:
What's the mongodb version on production? I don't get an array locally

Reply:
it's 4.x but I don't think that matters

I think the reason is that it's not an array to begin with
```
> db.test.insert({myArray: []})
WriteResult({ "nInserted" : 1 })
> db.test.update({},{$set:{'myArray.0': true, 'myArray.1': true}})
WriteResult({ "nMatched" : 1, "nUpserted" : 0, "nModified" : 1 })
> db.test.find()
{ "_id" : ObjectId("5f6caca6084a9ed68ff5f162"), "myArray" : [ true, true ] }
```

same code results in an object if `myArray` is not initially set to `[]`

Reply:
Should default `exhausted` it to an array during `channel` creation?

Reply:
I would suggest that you do that in the same piece of code that sets `created`. Would this require too many changes to the tests?

Reply:
It only required change to one test